tools/android/packaging/gradle.properties

```diff
@@ -0,0 +1,2 @@
+android.useAndroidX=true
+org.gradle.jvmargs=-Xmx4608m
```

Comment:
This is maximum memory usage right? It seems like a strange value?

Reply:
I guess he meant that 4608 is an unusual value, did you intend 4096m?

Reply:
Didn't really matter tbh, got me to run some further tests, down to smallest simple number (1024)